packages/client/modules/userDashboard/components/OrgUserRow/OrgMemberRow.tsx

```diff
@@ -173,16 +170,35 @@ const OrgMemberRow = (props: Props) => {
   )
 
   const {
-    user: {email, inactive, picture, preferredName, lastSeenAt},
+    user: {email, inactive, picture, preferredName, lastSeenAt, id: userId},
     role
   } = organizationUser
 
   const isBillingLeader = role === 'BILLING_LEADER'
   const isOrgAdmin = role === 'ORG_ADMIN'
+  const {isOrgAdmin: isViewerOrgAdmin} = organization
   const formattedLastSeenAt = lastSeenAt ? format(new Date(lastSeenAt), 'yyyy-MM-dd') : 'Never'
+  const isSelf = viewerId === userId
+  const canBeSelected = isViewerOrgAdmin && !isSelf && !isBillingLeader && !isOrgAdmin
```

Comment:
-1 it does not only look odd, but it also messes with the export function. I think we should allow selecting all users and then we have 2 options:
* disable the delete button with a tooltip: "Batch removing yourself is not supported"
* on remove click open a different dialog with an error message: "Cannot mass remove yourself"
* show the current dialog with a note: "You cannot batch remove yourself. Removing ${n-1} members" and just omitting the viewer from the list.

All these adapted for "Yourself, org admins or billing leaders".